functions/src/api/auth.js

```diff
@@ -0,0 +1,42 @@
+const admin = require("firebase-admin");
+const functions = require("firebase-functions");
+
+const { HttpsError } = require("firebase-functions/lib/providers/https");
+
+const { AuthConfigSchema } = require("../types/auth");
+
+const { checkIfValidVenueId } = require("../utils/venue");
+
+const getAuthConfigsCollectionRef = () =>
+  admin.firestore().collection("authConfigs");
+
+const getAuthConfigRef = (venueId) =>
+  getAuthConfigsCollectionRef().doc(venueId);
+
+// TODO: do we need to implement finding the sovereign venue to implement this properly..? Probably should..
```

Comment:
[TODO] do we need to implement finding the sovereign venue to implement this properly..? Probably should..

Reply:
Nope, so long as we have an `authConfig` on the main 'entrypoint' for the event, that is all we should need. Usually (if not always) that's the sovereign venue anyway.